packages/dd-trace/src/lambda/runtime/ritm.js

```diff
@@ -60,23 +59,18 @@ function _extractModuleNameAndHandlerPath (handler) {
 }
 
 /**
- * Returns the correct path of the file to be patched
- * when required.
+ * Returns all possible paths of the files to be patched when required.
  *
  * @param {*} lambdaStylePath the path comprised of the `LAMBDA_TASK_ROOT`,
  * the root of the module of the Lambda handler, and the module name.
- * @returns the lambdaStylePath with the appropiate extension for the hook.
+ * @returns the lambdaStylePath with appropiate extensions for the hook.
  */
-function _getLambdaFilePath (lambdaStylePath) {
-  let lambdaFilePath = lambdaStylePath
-  if (fs.existsSync(lambdaStylePath + '.js')) {
-    lambdaFilePath += '.js'
-  } else if (fs.existsSync(lambdaStylePath + '.mjs')) {
-    lambdaFilePath += '.mjs'
-  } else if (fs.existsSync(lambdaStylePath + '.cjs')) {
-    lambdaFilePath += '.cjs'
-  }
-  return lambdaFilePath
+function _getLambdaFilePaths (lambdaStylePath) {
+  return [
+    `${lambdaStylePath}.js`,
+    `${lambdaStylePath}.mjs`,
+    `${lambdaStylePath}.cjs`
+  ]
```

Comment:
So normally, customers would only have one of this, is it OK to just leave it as is rn?

Reply:
Yes, it will register a hook for all paths, and only the one that is actually used will hit.